### ISSUE TITLE
Serializing as Json produces invalid Json for numeric types depending on culture

### ISSUE THREAD

**Issue by ManelBH** [Microsoft/OpenAPI.NET#396]
If the CurrentCulture uses comma separators for decimals the json serializer will produce a json with invalid comma separated numerics (e.g. maximum and minimum fields in schema) It should produce a valid json regardless of culture.

**Comment by ManelBH** [comment]
A concrete example:

```
var stream = new HttpClient().GetStreamAsync("https://petstore.swagger.io/v2/swagger.json").Result;
var openApiDocument = new OpenApiStreamReader().Read(stream , out var diagnostic);
var outputString = openApiDocument.SerializeAsJson(OpenApiSpecVersion.OpenApi2_0);

Assert.Equal(new CultureInfo("es-ES"), CultureInfo.CurrentCulture);  //Ok

var jobject = JObject.Parse(outputString); //Throws JsonReaderException
```

Exception message: 

> Invalid JavaScript property identifier character: ,. Path 'paths./store/order/{orderId}.get.parameters[0].maximum', line 477, position 27.

Json string section:
```
...
"/store/order/{orderId}": {
      "get": {
        "tags": [
          "store"
        ],
        "summary": "Find purchase order by ID",
        "description": "For valid response try integer IDs with value >= 1 and <= 10. Other values will generated exceptions",
        "operationId": "getOrderById",
        "produces": [
          "application/xml",
          "application/json"
        ],
        "parameters": [
          {
            "in": "path",
            "name": "orderId",
            "description": "ID of pet that needs to be fetched",
            "required": true,
            "type": "integer",
            "format": "int64",
            "maximum": 10,0,
            "minimum": 1,0
          }
        ],
...
```

**Comment by darrelmiller** [comment]
I believe this issue is fixed with the PR #385 We just need to get a new release out.

**Comment by darrelmiller** [comment]
Fixed in 1.1.3
